src/community/roadmap/index.md.njk

```diff
@@ -28,24 +28,25 @@ We recently:
 - released [GOV.UK Frontend v4.0.1](https://github.com/alphagov/govuk-frontend/releases/tag/v4.0.1)
 - updated the [Accessible autocomplete](https://github.com/alphagov/accessible-autocomplete/releases/tag/v2.0.4) component and clarified [future plan](https://github.com/alphagov/accessible-autocomplete/issues/532)
 - made changes to [Start page](https://github.com/alphagov/design-system-team-internal/issues/552) pattern
+- released [GOV.UK Frontend v4.1.0](https://github.com/alphagov/govuk-frontend/releases/tag/v4.1.0)
+- published the help users to [Confirm a phone number](https://github.com/alphagov/govuk-design-system-backlog/issues/239) pattern
+- released improvements to the screen reader experience for the character count component
```

Comment:
Is it worth merging the 4.1.0 and the character count bullet? There's a lot of frontend releases in this list and I think it makes it a little more meaningful if we provide info on what's actually in those releases.

```suggestion
- released [GOV.UK Frontend v4.1.0](https://github.com/alphagov/govuk-frontend/releases/tag/v4.1.0) with improvements to the screen reader experience for the character count component
- published the help users to [Confirm a phone number](https://github.com/alphagov/govuk-design-system-backlog/issues/239) pattern
```